modules/ROOT/pages/debugging-kernel-crashes.adoc

```diff
@@ -0,0 +1,32 @@
+= Enabling kdump to debug kernel crashes
+
+. Memory must be reserved for the crash kernel during booting of the first kernel. Kernel arguments can be provided like this:
++
+[source, bash]
+----
+sudo rpm-ostree kargs --append='crashkernel=256M'
```

Comment:
The doc suggests 128, maybe we should keep that 'default'? Unless there is a reason fcos needs more memory.

https://fedoraproject.org/wiki/How_to_use_kdump_to_debug_kernel_crashes

Reply:
Based off of a local qemu build of the current `testing-devel`, it seems like 128M does not work.